reviews/github/madler/zlib/trail-of-bits-2016.md

```diff
@@ -0,0 +1,53 @@
+---
+Publication-State: Active
+Access: Public
+Reviewers:
+- Organization: Trail of Bits
+  Associated-With-Project: false
+  Compensation-Source: non-project
+- Organization: TrustInSoft
+  Associated-With-Project: false
+  Compensation-Source: non-project  
+Domain: Security
+Methodology:
+- External-Review
+Issues-Identified: Non-Severe
+Package-URLs:
+- pkg:github/madler/zlib@1.2.8
+Review-Date: 2016-09-30
+Scope: Implementation/Partial
+Schema-Version: 1.0
+SPDX-License-Identifier: CC-BY-4.0
+---
+
+### Summary
+
+Five security issues were identified (one medium-risk, four low-risk), all related to undefined behavior. All but one (low risk) issue was fixed before the report was published in September 2016.
```

Comment:
This PR looks good to me! Marking it as approved, but as a small change do want to include a recommended fix that is easily viewable for the user in the summary?

Maybe include a sentence in the end saying "it is recommended to update to version 1.2.11 to resolve these four issues." See this GitHub PR from OSSEC for reference: https://github.com/ossec/ossec-hids/pull/1198.

Reply:
Thanks! Added a note to upgrade to 1.2.11 in the summary and clarified the wording about the external fix, but I think we should keep the actual code within the external report.